pypeit/spectrographs/spectrograph.py

```diff
@@ -573,8 +578,15 @@ def get_meta_value(self, inp, meta_key, required=False, ignore_bad_header=False,
                 value = headarr[self.meta[meta_key]['ext']][self.meta[meta_key]['card']]
             except (KeyError, TypeError):
                 value = None
+        # Return now?
```

Comment:
Can you explain why we need this option?

Reply:
It's not elegant.  But if you look a few lines down, you see 
the calls to `get_meta_value()` for `ra,dec` that use this. 

Without this option, the crash gets into an infinite loop...